src/gui/statusbar.cpp

```diff
@@ -129,14 +132,21 @@ StatusBar::StatusBar(QWidget *parent)
 #ifndef Q_OS_MACOS
     statusSep4->setFrameShadow(QFrame::Raised);
 #endif
-    layout->addWidget(m_DHTLbl);
+    QFrame *statusSep5 = new QFrame(this);
```

Comment:
```suggestion
    auto *statusSep5 = new QFrame(this);
```

Reply:
All of the other separators using the explicit `QFrame` type

Reply:
I'm ok with the full type. Using `auto` is also OK for our case here.